app/themes/dashboard/ui/index.html

```diff
@@ -0,0 +1,116 @@
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width">
+    <link rel="shortcut icon" href="/images/favicon.ico" />
+
+    <!-- build:css styles/lib.css -->
+    <!-- bower:css -->
+    <!-- endbower -->
+    <!-- endbuild -->
+
+    <!-- build:css styles/app.css -->
+    <!-- inject:css -->
+    <!-- endinject -->
+    <!-- endbuild -->
+
+  </head>
+  <body ng-app="app" class="skin-black">
+
+    <div class="wrapper">
+      <header class="main-header">
+
+        <!-- Logo -->
+        <a ui-sref="root.landing" class="logo">
+          <!-- mini logo for sidebar mini 50x50 pixels -->
+          <span class="logo-mini"><b>ML</b></span>
+          <!-- logo for regular state and mobile devices -->
+          <span class="logo-lg"><h1 id="logo">Sample App</h1></span>
+        </a>
+
+        <nav class="navbar navbar-static-top" role="navigation" ng-controller="topnavCtrl as topnav" ng-cloak>
+          <!-- Navbar Right Menu -->
+          <div class="navbar-custom-menu">
+            <ul class="nav navbar-nav" ng-if="!topnav.currentUser">
+              <li class="user">
+                <a ui-sref="root.login"><span class="glyphicon glyphicon-user"></span> Sign In</a></li>
+            </ul>
+            <ul class="nav navbar-nav" ng-if="topnav.currentUser">
+              <li>
+                <a ui-sref="root.landing">
+                  <i class="fa fa-dashboard"></i>
+                  Dashboard
+                </a>
+              </li>
+              <li>
+                <a ui-sref="root.search">
+                  <i class="fa fa-search"></i>
+                  Search
+                </a>
+              </li>
+              <li class="dropdown user user-menu">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                  <i class="fa fa-user"></i>
+                  <span class="hidden-xs">{{ topnav.currentUser.name }}</span>
+                </a>
+                <ul class="dropdown-menu">
+                  <!-- User image -->
+                  <li class="user-header">
+                    <p>
+                      {{ topnav.currentUser.name }}
+                    </p>
+                  </li>
+                  <!-- Menu Footer-->
+                  <li class="user-footer">
+                    <div class="pull-left">
+                      <a ui-sref="root.profile" class="btn btn-default btn-flat">Profile</a>
+                    </div>
+                    <div class="pull-right">
+                      <a ng-click="topnav.logout()" class="btn btn-default btn-flat">Sign out</a>
+                    </div>
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </div>
+
+        </nav>
+      </header>
+
+      <ui-view></ui-view>
+
+
+      <footer id="footer" class="powered" ng-controller="FooterCtrl as footer">
+        <div class="container">
+          <div class="row">
```

Comment:
I see what you mean now. There's all the section with the icons that are missing?